vllm/config.py

```diff
@@ -192,7 +192,7 @@ def _verify_quantization(self) -> None:
                 logger.warning(
                     "%s quantization is not fully "
                     "optimized yet. The speed can be slower than "
-                    "non-quantized models.", self.quantization)
+                    "non-quantized models.", self.quantization, extra=logger_data)
```

Comment:
`logger_data` is undefined here, I think

Reply:
Thanks, fixed!